lib/brains/views/customer.py

```diff
@@ -22,38 +19,21 @@ def create(request):
     if not form.is_valid():
         return Response(form.errors, status=400)
 
-    uuid = form.cleaned_data['uuid']
-
-    # Create the solitude objects.
-    buyer, buyer_created = Buyer.objects.get_or_create(uuid=uuid)
-    log.info('Buyer {0}: {1}'
-             .format('created' if buyer_created else 'exists', buyer.pk))
-
-    braintree, braintree_buyer_created = (
-        BraintreeBuyer.objects.get_or_create(buyer=buyer))
-    log.info('BraintreeBuyer {0}: {1}'
-             .format('created' if braintree_buyer_created else 'exists',
-                     braintree.pk))
-
-    braintree_id = braintree.braintree_id
-    # Create the braintree objects.
-    # If anything fails at this point, the transaction will be rolled back.
-    braintree_created = False
-    try:
-        customer = client.find(braintree_id)
-        log.info('Customer found: {0}'.format(braintree_id))
-    except NotFoundError:
-        result = client.create({'id': braintree_id})
-        if not result.is_success:
-            log.warning('Error on creating Customer: {0}, {1}'
-                        .format(uuid, result.message))
-            raise BraintreeResultError(result)
-
-        braintree_created = True
-        customer = result.customer
-
-    res = BuyerSerializer(instance=buyer).data
-    res.update({'braintree': serializers.Customer(instance=customer).data})
-    created = any([braintree_created, braintree_buyer_created, buyer_created])
-    status = 201 if created else 200
-    return Response(res, status)
+    result = client.create()
+    if not result.is_success:
+        log.warning('Error on creating Customer: {0}, {1}'
+                    .format(form.cleaned_data['uuid'], result.message))
+        raise BraintreeResultError(result)
+
+    log.info('Braintree customer created: {0}'.format(result.customer.id))
+
+    braintree_buyer = BraintreeBuyer.objects.create(
+        buyer=form.buyer, braintree_id=result.customer.id)
+
+    log.info('Braintree buyer created: {0}'.format(braintree_buyer.pk))
+
+    res = serializers.Namespaced(
+        serializers.LocalBuyer(instance=braintree_buyer),
```

Comment:
I think it would be nicer to read this code as:
````
res = serializers.Namespaced(
    braintree=...,
    mozilla=...,
)
````
You can pass keyword args for positional args so it will work.